onmt/translate/decode_strategy.py

```diff
@@ -84,6 +79,19 @@ def __init__(self, pad, bos, eos, batch_size, device, parallel_paths,
 
         self.done = False
 
+    def initialize(self, device):
```

Comment:
Last comment: the base method should have the same interface as the child classes, even if some arguments are unused.